website/client/components/challenges/leaveChallengeModal.vue

```diff
@@ -0,0 +1,44 @@
+<template lang="pug">
+  b-modal#leave-challenge-modal(title="Leave Challenge", size='sm', :hide-footer="true")
```

Comment:
This will need to be translatable, in the end. Try

`:title='$t("leaveChallenge")'`

Although it looks like you might be thinking of having that in a heading in the modal body ("leaveCha"?), in which case you wouldn't need a title attribute at all.

Reply:
I am using a header already, so I guess I don't need a title attr.